{"uuid": "7c89fb1c-954b-49cd-a944-a4ff616d3e5c", "children": ["dafddea7-64f6-4df5-bdf1-f7dba1450e38"], "befores": [{"name": "setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.111\n  (Session info: chrome=116.0.5845.110)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00007FF7D8F23E62+57250]\n\t(No symbol) [0x00007FF7D8E9BC02]\n\t(No symbol) [0x00007FF7D8D6E0BB]\n\t(No symbol) [0x00007FF7D8D5BA2A]\n\t(No symbol) [0x00007FF7D8D5B973]\n\t(No symbol) [0x00007FF7D8D5A3F5]\n\t(No symbol) [0x00007FF7D8D5AD2E]\n\t(No symbol) [0x00007FF7D8D67DCB]\n\t(No symbol) [0x00007FF7D8D77C51]\n\t(No symbol) [0x00007FF7D8D7BDEA]\n\t(No symbol) [0x00007FF7D8D5B2F7]\n\t(No symbol) [0x00007FF7D8D77B2A]\n\t(No symbol) [0x00007FF7D8DDE379]\n\t(No symbol) [0x00007FF7D8DC6583]\n\t(No symbol) [0x00007FF7D8D9CC71]\n\t(No symbol) [0x00007FF7D8D9DE54]\n\tGetHandleVerifier [0x00007FF7D91D4CF2+2879026]\n\tGetHandleVerifier [0x00007FF7D9226F30+3215472]\n\tGetHandleVerifier [0x00007FF7D921FD4F+3186319]\n\tGetHandleVerifier [0x00007FF7D8FB5505+652869]\n\t(No symbol) [0x00007FF7D8EA7518]\n\t(No symbol) [0x00007FF7D8EA35F4]\n\t(No symbol) [0x00007FF7D8EA36EC]\n\t(No symbol) [0x00007FF7D8E938E3]\n\tBaseThreadInitThunk [0x00007FFA86F47614+20]\n\tRtlUserThreadStart [0x00007FFA872426B1+33]\n\n", "trace": "  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 901, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\conftest.py\", line 41, in setup\n    get_webdriver.get(URL)\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 449, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Owner\\PycharmProjects\\LifeHelp\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1692942148194, "stop": 1692942448208}], "start": 1692942148194, "stop": 1692942448339}